includes/blocks/correction-box-block/site-editor.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { Path, SVG, Placeholder } from '@wordpress/components';
```

Comment:
I'd rather not using the `Placeholder` component

`import { Path, SVG } from '@wordpress/components';`